### ISSUE TITLE
Measure along / within ROIs

### ISSUE THREAD

**Issue by half-adder** [matplotlib/napari-matplotlib#45]
Not sure if this is within the scope of the project, but it would be nice to be able to measure along ROIs (e.g. a "profile line" through that moves through space in one (more, eventually?) axis. Through lines, it could be a profile line. With polygons, it could be average intensity (or any other function that takes a set of pixels and returns a number), or a profile line along the circumference of the shape.

Something like this as an example of the profile line:
![image](https://user-images.githubusercontent.com/10676434/167321868-b2b792f2-2097-433e-a87d-a7927a3907e3.png)

**Comment by dstansby** [comment]
Yes, this is definitely within scope - if someone wants to have a go and open a PR it would be very welcome! 😄
